### PR TITLE
Add UPnP Troubleshooting

### PR DESCRIPTION
**Pull request by argakiig**
Update troubleshooting with UPnP specific callouts
image

### CODE REVIEW

**Comment by zhyatt** [comment]
@argakiig This is looking good. Two questions:

1. What terms do you think someone would use to search for this issue? We should make sure those are included someone in the section - for instance "networking" or something
1. Do we need to link back to this section from anywhere else in the docs?

File: docs/running-a-node/troubleshooting.md [hunk 1]
```diff
@@ -10,6 +10,45 @@ The default location of standard node log files for various systems:
...
+
+### Enable UPnP logging
+Appending this to your launch command will enable upnp logging.
+```bash
+--config node.logging.upnp_details=true
+```
+### Check Logs for responses
+Check the begginning of the logs for UPNP_* messages
+
+!!! warning "Standard message indcating no existing mapping"
```
Comment (guilhermelawless) [inline docs/running-a-node/troubleshooting.md]: So when I get this message it's working as intended, and when the UPnP loops again this message will not be there as the mapping was created? If so, as a user this message reads like an error, and maybe we should either:

1. Make a big notice here that it is the intended behavior, or
2. In the node, log differently when the error code is 714 to make it look less like an error

I like 2 more, as the user will not need to check the documentation, but could go with either.

Also small typo, ind_i_cating

File: docs/running-a-node/troubleshooting.md [hunk 2]
```diff
@@ -10,6 +10,45 @@ The default location of standard node log files for various systems:
 
 ---
 
+## Troubleshooting UPnP
+
+### Ensure UPnP is enabled
+UPnP will be enabled unless the external port is set in either the config
+```bash
```
Comment (guilhermelawless) [inline docs/running-a-node/troubleshooting.md]: Could use toml code here

File: docs/running-a-node/troubleshooting.md [hunk 3]
```diff
@@ -10,6 +10,45 @@ The default location of standard node log files for various systems:
 
 ---
 
+## Troubleshooting UPnP
```
Comment (guilhermelawless) [inline docs/running-a-node/troubleshooting.md]: I think the "What to do if the node crashes (Linux)" section should stay higher

**Comment by guilhermelawless** [comment]
As @zhyatt said, maybe we could make a quick reference to this troubleshooting UPnP page around [here](https://docs.nano.org/running-a-node/node-setup/#network-ports). Specifically if they have a restrictive firewall or UPnP disabled they might want to check the page.

**Pull request merged**